### PR TITLE
added document_ftp

### PR DESCRIPTION
**Pull request by cmsalmeida**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please remove .bzr directory

**Comment by cmsalmeida** [comment]
.bzr directory removed

File: document_ftp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*.py[cod]
+*~
+
+.bzr
```
Comment (dreispt) [inline document_ftp/.gitignore:4]: remove file

File: document_ftp/__init__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#    
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>).
```
Comment (dreispt) [inline document_ftp/__init__.py:5]: Put the author's (you/your company) name here instead

File: document_ftp/__init__.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.     
+#
+##############################################################################
+
+import ftpserver
+import wizard
+import res_config
```
Comment (dreispt) [inline document_ftp/__init__.py:24]: for module files use `from . import X`

File: document_ftp/__init__.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+post_load = ftpserver.start_server
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (dreispt) [inline document_ftp/__init__.py:28]: remove vim lines

File: document_ftp/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Shared Repositories (FTP)',
+    'version': '1.004',
```
Comment (dreispt) [inline document_ftp/__openerp__.py:25]: `8.0.x.y.z`

File: document_ftp/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    'category': 'Knowledge Management',
```
Comment (dreispt) [inline document_ftp/__openerp__.py:26]: Could OCA\knowledge be a better home for this module?

File: document_ftp/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    'description': """
```
Comment (dreispt) [inline document_ftp/__openerp__.py:27]: Extract the description to a README.rst file

File: document_ftp/ftpserver/abstracted_fs.py [hunk 8]
```diff
@@ -0,0 +1,667 @@
+# -*- encoding: utf-8 -*-
+
```
Comment (dreispt) [inline document_ftp/ftpserver/abstracted_fs.py:2]: Is this your own work, or is is copyrighted by someone else?

File: document_ftp/__openerp__.py (hunk 6 above)
Comment (cmsalmeida) [inline document_ftp/__openerp__.py:26]: This is the OpenERP document_ftp migration, what you suggest me to do?

File: document_ftp/ftpserver/abstracted_fs.py (hunk 8 above)
Comment (cmsalmeida) [inline document_ftp/ftpserver/abstracted_fs.py:2]: As I said this if the OpenERP document_ftp, working now on 8.0

Comment (dreispt) [inline document_ftp/ftpserver/abstracted_fs.py:2]: Sorry for not noticing that. Then Openerp SA copyright should be left in the code files and in the manifest's list of authors.
A PEP8/style cleanup should be done, nevertheless.

File: document_ftp/__openerp__.py (hunk 6 above)
Comment (dreispt) [inline document_ftp/__openerp__.py:26]: If it's a generic FTP server, it's best to host it here.

**Comment by angelmoya** [comment]
Error appears on installation, maybe you chaged name of this module for internal purpose

ValueError: External ID not found in the system: tko_document_ftp.action_document_browse

**Comment by angelmoya** [comment]
+1 to move this module to OCA/knowledge

**Comment by cmsalmeida** [comment]
Yes it was done while in migration process to follow our infrastructure patterns. I will aks to fix and then PR to knowledge
Thank you!

**Comment by cmsalmeida** [comment]
moved to OCA/knowledge

**Pull request closed without merging**